Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -94,13 +105,31 @@ protected override bool IsValidController(IMixedRealityController controller)
                             }
                         }
                     }
+
+                    // Check if the palm angle meets the prescribed threshold
+                    palmFacingThresholdMet = palmCameraAngle < facingThreshold;
```

Comment:
Interesting, I just had the thought that this could be made more general since currently the logic is "if the palm faces the camera, follow the camera, otherwise follow the tracked object". I think it could be more general by saying "if the tracked object faces the camera, follow the camera, otherwise follow the tracked object". Then it could work for things like motion controllers, other joints, or even following other game objects.

Not required for this PR, but was just interesting.

Reply:
I agree that it could work for more than just hands!

Really it's a matter of checking the SolverHandler.TargetTransform - that said, facing the camera for a palm means it's up direction aligns with the gaze vector. 
For a general purpose Solver, it's forward might be more useful. It's difficult to generalize that.